### PR TITLE
Create cvp-common-aws-osd workflow

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject

### CODE REVIEW

File: ci-operator/step-registry/optional-operators/cvp-common/aws-osd/optional-operators-cvp-common-aws-osd-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+workflow:
+  as: optional-operators-cvp-common-aws-osd
+  steps:
+    cluster_profile: aws-osd-msp
+    env:
+      #BASE_DOMAIN: cpaas-ci.devcluster.openshift.com
```
Comment (dirgim) [inline ci-operator/step-registry/optional-operators/cvp-common/aws-osd/optional-operators-cvp-common-aws-osd-workflow.yaml:6]: Just a question - how is the base domain set for the MSP OSD cluster profile?

**Comment by jsztuka** [comment]
Is the purpose of this workflow the same as for:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/optional-operators/cvp-common/aws/optional-operators-cvp-common-aws-workflow.yaml
Meaning it will not use cluster claim?

**Comment by samvarankashyap** [comment]
@jsztuka yes, we would not be using the clusterpools. We would be provisioning using the osd service.

**Review by jsztuka — APPROVED** [review]
LGTM

File: ci-operator/step-registry/optional-operators/cvp-common/aws-osd/optional-operators-cvp-common-aws-osd-workflow.yaml (hunk 1 above)
Comment (jsztuka) [reply ci-operator/step-registry/optional-operators/cvp-common/aws-osd/optional-operators-cvp-common-aws-osd-workflow.yaml:6]: I would like to know also.

Comment (samvarankashyap) [reply ci-operator/step-registry/optional-operators/cvp-common/aws-osd/optional-operators-cvp-common-aws-osd-workflow.yaml:6]: Hi @dirgim 
we have a way to configure a https://docs.openshift.com/dedicated/applications/deployments/osd-config-custom-domains-applications.html 
i think that should help. 
I am still figuring out if the custom domains work, and where to get the certificates from.

**Pull request merged**